src/Stratis.Bitcoin.Features.RPC/IPAddressBlock.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Net;
+using NBitcoin;
+
+namespace Stratis.Bitcoin.Features.RPC
+{
+    /// <summary>
+    /// The class of an object representing a block of ip addresses.
+    /// </summary>
+    public class IPAddressBlock
+    {
+        public int Mask { get; private set; }
+        public IPAddress Address { get; private set; }
+
+        public int FullMask => 8 * this.Address.GetAddressBytes().Length;
```

Comment:
FullMaskLengthInBytes (or add a comment that this is a length of a mask in bytes)

Reply:
Renamed to `FullMaskLengthInBits`.